src/main/java/edu/iis/powp/adapter/PlotterMagicToDrawerAdapter.java

```diff
@@ -18,7 +18,6 @@
 	
 	public PlotterMagicToDrawerAdapter() {
 		super();
-		dpc = ApplicationWithDrawer.getDrawPanelController();		
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with ApplicationWithDrawer class.